File: Source/Classes/NCCU/HistoryCellTableViewCell.m
//
//  HistoryCellTableViewCell.m
//  Mumble
//
//  Created by HanChien Chun on 2016/8/24.
//
//

#import "HistoryCellTableViewCell.h"

@implementation HistoryCellTableViewCell

- (void)awakeFromNib {
    [super awakeFromNib];
    // Initialization code
}

- (void)setSelected:(BOOL)selected animated:(BOOL)animated {
    [super setSelected:selected animated:animated];

    // Configure the view for the selected state
}

- (void)dealloc {
    [_rating release];
    [_labelDuration release];
    [_labelExpert release];
    [_labelDate release];
    [super dealloc];
}
@end

File: Source/Classes/NCCU/CallHistory/HistoryCellTableViewCell.swift
//
//  HistoryCellTableViewCell.swift
//  Mumble
//
//  Created by William on 2017/4/27.
//
//

import UIKit
import HCSStarRatingView

class HistoryCellTableViewCell: UITableViewCell {
    
    @IBOutlet var rating: HCSStarRatingView!
    @IBOutlet var labelDuration: UILabel!
    @IBOutlet var labelExpert: UILabel!
    @IBOutlet var labelDate: UILabel!
    
    override func awakeFromNib() {
        super.awakeFromNib()
        // Initialization code
    }
    
    override func setSelected(_ selected: Bool, animated: Bool) {
        super.setSelected(selected, animated: animated)
        // Configure the view for the selected state
    }
    
    deinit {
    }
    
}


